experimental-features-4.0.md

```diff
@@ -1,23 +1,40 @@
 ---
-title: TiDB 4.0 实验特性
-summary: 了解 TiDB 4.0 版本的实验特性。
+title: TiDB 实验特性
+summary: 了解 TiDB 所有版本的实验特性。
```

Comment:
Do we also need to change the file name?

Reply:
Yes, please change the file name to experimental-features.md. @scsldb

Reply:
done